tests/apm_tracing_e2e/test_otel.py

```diff
@@ -38,7 +38,7 @@ def test_datadog_otel_span(self):
         assert child.get("parentID") == parent.get("spanID")
         assert child.get("spanID") != "10000"
         assert child.get("duration") == "1000000000"
-        assert child.get("type") == "internal"
+        assert child.get("meta").get("span.kind") == "internal"
```

Comment:
You should annotate this method with @missing_feature for current golang prod release and for Java (see CI failures).